### ISSUE TITLE
publish wheels for windows?

### ISSUE THREAD

**Issue by balopat** [sbdchd/codeowners#5]
Is there a plan to publish wheels to windows?
Currently windows  `pip install` fails: 

```
Collecting codeowners
  Downloading https://files.pythonhosted.org/packages/7f/42/41721a58be6b31ee72c84dfb5eebabac2a36908f789b92ab31615b489152/codeowners-0.1.2.tar.gz (9.1MB)
     |████████████████████████████████| 9.1MB ...
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... error
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\me\envs\tryinstall\scripts\python.exe' 'c:\users\me\envs\tryinstall\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\,e\AppData\Local\Temp\tmpw35jh_ky'
         cwd: C:\Users\me\AppData\Local\Temp\pip-install-4uwcouev\codeowners
    Complete output (16 lines):
    Traceback (most recent call last):
      File "c:\users\me\envs\tryinstall\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 207, in <module>
        main()
      File "c:\users\me\envs\tryinstall\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 197, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "c:\users\me\envs\tryinstall\lib\site-packages\pip\_vendor\pep517\_in_process.py", line 69, in prepare_metadata_for_build_wheel
        return hook(metadata_directory, config_settings)
      File "C:\Users\me\AppData\Local\Temp\pip-build-env-jqhntcdm\overlay\Lib\site-packages\poetry\core\masonry\api.py", line 35, in prepare_metadata_for_build_wheel
        builder = WheelBuilder(poetry)
      File "C:\Users\me\AppData\Local\Temp\pip-build-env-jqhntcdm\overlay\Lib\site-packages\poetry\core\masonry\builders\wheel.py", line 46, in __init__
        super(WheelBuilder, self).__init__(poetry, executable=executable)
      File "C:\Users\me\AppData\Local\Temp\pip-build-env-jqhntcdm\overlay\Lib\site-packages\poetry\core\masonry\builders\builder.py", line 86, in __init__
        includes=includes,
      File "C:\Users\me\AppData\Local\Temp\pip-build-env-jqhntcdm\overlay\Lib\site-packages\poetry\core\masonry\utils\module.py", line 59, in __init__
        "No file/folder found for package {}".format(name)
    poetry.core.masonry.utils.module.ModuleOrPackageNotFound: No file/folder found for package codeowners
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\me\envs\tryinstall\scripts\python.exe' 'c:\users\me\envs\tryinstall\lib\site-packages\pip\_vendor\pep517\_in_process.py' prepare_metadata_for_build_wheel 'C:\Users\me\AppData\Local\Temp\tmpw35jh_ky' Check the logs for full command output.
```

**Comment by sbdchd** [comment]
TBH, probably not. I don't use windows for dev, and I don't need/use this library anymore

**Comment by sbdchd** [comment]
Should be fixed in latest release, 0.2.0

Let me know if you have any issues
